src/group/mls_group/mod.rs

```diff
@@ -470,6 +470,10 @@ impl MlsGroup {
 }
 
 // Callback functions
+
+/// This callback function is used in several places in `MlsGroup`.
```

Comment:
So when we land this now we should add a TODO and an issue number to make clear that this is probably going away again.